sklearn/model_selection/_validation.py

```diff
@@ -169,6 +170,11 @@ def cross_validate(
 
         .. versionadded:: 0.20
 
+    return_indices : bool, default=False
+        Whether to return the train-test indices selected for each split.
```

Comment:
Coming from https://github.com/scikit-learn/scikit-learn/discussions/21664, I agree `return_indices` is useful. (I wanted to do something like this recently).